npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater_spec.rb

```diff
@@ -127,63 +127,8 @@
         expect(updated_fetch_factory_version).to eq("0.0.2")
       end
     end
-  end
-
-  describe "errors" do
-    context "with a sub dependency name that can't be found" do
-      let(:files) { project_dependency_files("npm6/github_sub_dependency_name_missing") }
-
-      let(:dependency_name) { "test-missing-dep-name-npm-package" }
-      let(:requirements) do
-        [{
-          requirement: nil,
-          file: "package.json",
-          groups: ["dependencies"],
-          source: {
-            type: "git",
-            url: "https://github.com/dependabot-fixtures/"\
-            "test-missing-dep-name-npm-package",
-            branch: nil,
-            ref: ref
-          }
-        }]
-      end
-      let(:previous_requirements) do
-        [{
-          requirement: nil,
-          file: "package.json",
-          groups: ["dependencies"],
-          source: {
-            type: "git",
-            url: "https://github.com/dependabot-fixtures/"\
-            "test-missing-dep-name-npm-package",
-            branch: nil,
-            ref: old_ref
-          }
-        }]
-      end
-      let(:previous_version) { "1be88e036981a8511eacf3f20e0a21507349988d" }
-      let(:version) { "346e79a12f34c5937bf0f016bced0723f864fe19" }
-      let(:ref) { "v1.0.1" }
-      let(:old_ref) { "v1.0.0" }
-
-      it "raises a helpful error" do
-        expect { updated_npm_lock_content }.
-          to raise_error(Dependabot::DependencyFileNotResolvable)
-      end
-    end
-
-    context "with an invalid requirement in the package.json" do
-      let(:files) { project_dependency_files("npm6/invalid_requirement") }
-
-      let(:npm_lock_fixture_name) { "package-lock.json" }
-
-      it "raises a helpful error" do
-        expect { updated_npm_lock_content }.
-          to raise_error(Dependabot::DependencyFileNotResolvable)
-      end
-    end
 
+    # NOTE: This no longer raises in npm 7
     context "when there is a private git dep we don't have access to" do
```

Comment:
No longer fails for npm 6, tests have moved here: https://github.com/dependabot/dependabot-core/blob/bfbc137cddb2da65f0e37f6ee4fae8c6dee75067/npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater_spec.rb#L131-L170